src/components/DayPickerNavigation.jsx

```diff
@@ -74,39 +89,49 @@ export default function DayPickerNavigation(props) {
   });
   const prevClassNames = cx('DayPickerNavigation__prev', {
     'DayPickerNavigation__prev--default': isDefaultNavPrev,
+    'DayPickerNavigation__prev--rtl': isRTL,
   });
   const nextClassNames = cx('DayPickerNavigation__next', {
     'DayPickerNavigation__next--default': isDefaultNavNext,
+    'DayPickerNavigation__next--rtl': isRTL,
   });
 
+
+  const prevButton = () => (
```

Comment:
this will be a bit verbose imo, what do you suggest? keep in mind that the order of rendering here is important (that's why i refactored it to a function to avoid duplicate code)